app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :update_like_count
+
+  private
+
+  def update_like_count
+    increment!(:likes_counter)
```

Comment:
- [ ] It would be better if we improve our `update_like_count` method for accurate results, here instead of `increment!(:likes_counter)` let's write `post.increment!(:likes_counter)` so that we can increment the likes on the post